src/config/kpis.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'token' => env('KPI_TOKEN'),
```

Comment:
Please add a comment that explains what the token is for.

Reply:
Also, my (undocumented) env variable naming convention is to use `KPIS_TOKEN` for a config variable `kpis.token`.

```suggestion
    'token' => env('KPIS_TOKEN'),
```